sa4u_z3/main.py

```diff
@@ -382,6 +383,9 @@ def HUP_signal_handler(sig_num: int, _frame):
         pass
     print("In HUP signal handler...", flush=True)
 
+def TERM_signal_handler(sig_num: int, _frame):
```

Comment:
Per [PEP 8](https://peps.python.org/pep-0008/) there should be 2 spaces between top-level functions

Reply:
Fixed